homeassistant/components/sensor/fritzbox_netmonitor.py

```diff
@@ -0,0 +1,146 @@
+"""
+Support for monitoring the local system.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.systemmonitor/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_HOST, CONF_RESOURCES, CONF_TYPE)
+from homeassistant.components import group
+from homeassistant.helpers.entity import Entity
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['fritzconnection==0.6']
+
+GROUP_NAME_ALL_DEVICES = 'fritz status'
+ENTITY_ID_ALL_DEVICES = group.ENTITY_ID_FORMAT.format('fritz_status')
+
+CONF_DEFAULT_IP = '169.254.1.1'  # This IP is valid for all FRITZ!Box routers.
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_HOST, default=CONF_DEFAULT_IP): cv.string,
+})
+
+_LOGGER = logging.getLogger(__name__)
+
+SENSOR_TYPES = {
+    'is_linked': ['Link active', '', 'mdi:web'],
```

Comment:
This is a true/false sensor type and thus should be moved to a binary_sensor. There are probably a bunch more too.